functions/problem-set-functions.md

```diff
@@ -0,0 +1,243 @@
+# Problem Set: Functions
```

Comment:
(low priority)

Throughout, I suggest changing capitalized Truthy and Falsy to truthy and falsy, to make it less Pythonic. (unless it should be capitalized due to starting a sentence)

Reply:
Did this change!